config/linux.config

```diff
@@ -48,13 +48,20 @@ if target_arch in [Architecture.ARM, Architecture.ARMv7]:
 if target_arch_flags is not None:
     arch_flags += ' %s ' % (target_arch_flags)
 
-if host is None and target_arch != arch:
-    host = _host
+host_was_specified_before = 'true'
```

Comment:
It can be simplified. Before doing anything with the host variable store if we need to skip the tools prefix:

skip_tools_prefix = host is None and target_arch not in [Architecture.X86, Architecture.X86_64]